platform/api/src/database/mod.rs

```diff
@@ -8,12 +8,12 @@ mod session;
 mod ulid;
 mod user;
 
+pub use self::ulid::*;
```

Comment:
Do you need to have self here?

Reply:
Yeah, it wouldn't compile otherwise because it doens't know if you mean the ulid crate or module